build_from.py

```diff
@@ -95,5 +143,13 @@ def generate_diffs(reg_tree, act_title_and_section, builder, layer_cache):
     parser.add_argument('--checkpoint', required=False,
                         help='Directory to save checkpoint data')
 
+    parser.add_argument('--operation', action='store')
```

Comment:
Sure, this makes sense. As I noted elsewhere, the whole runner should really be refactored.